lighthouse-extension/app/src/popup.js

```diff
@@ -157,7 +166,10 @@ document.addEventListener('DOMContentLoaded', _ => {
 
       if (!multipleTabs && !debuggerExists) {
         feedbackEl.className = 'feedback-error';
-        feedbackEl.appendChild(buildReportErrorLink(err));
+        buildReportErrorLink(err)
```

Comment:
ideally we'd want the error passed to `background.console.error` below to also be source mapped